auth0/resource_auth0_log_stream.go

```diff
@@ -0,0 +1,366 @@
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"aws_account_id": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+							ForceNew:  true,
+						},
+						"aws_region": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+						},
+						"aws_partner_event_source": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Sensitive:   true,
+							Description: "Name of the Partner Event Source to be used with AWS, if the type is 'eventbridge'",
+						},
+						"azure_subscription_id": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+							ForceNew:  true,
+						},
+						"azure_resource_group": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+						},
+						"azure_region": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+						},
+						"azure_partner_topic": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Sensitive:   true,
+							ForceNew:    true,
+							Description: "Name of the Partner Topic to be used with Azure, if the type is 'eventgrid'",
+						},
+						"http_content_format": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"JSONLINES", "JSONARRAY"}, false),
+						},
+						"http_content_type": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "HTTP Content Type",
+						},
+						"http_endpoint": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "HTTP endpoint",
+						},
+						"http_authorization": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+						},
+						"http_custom_headers": {
+							Type:        schema.TypeSet,
+							Elem:        &schema.Schema{Type: schema.TypeString},
+							Optional:    true,
+							Description: "custom HTTP headers",
+						},
+
+						"datadog_region": {
+							Type:      schema.TypeString,
+							Sensitive: true,
+							Optional:  true,
+						},
+						"datadog_api_key": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+						},
+						"splunk_domain": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"splunk_token": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+						},
+						"splunk_port": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"splunk_secure": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func createLogStream(d *schema.ResourceData, m interface{}) error {
+	ls := expandLogStream(d)
+	fmt.Printf("CREATE: ---------------> %+v\n", ls)
```

Comment:
@mcalster consider using `TF_LOG=debug AUTH0_DEBUG=true` when running your tests instead of adding printf statements in your code.